apps/ios/Shared/ContentView.swift

```diff
@@ -48,28 +48,19 @@ struct ContentView: View {
                     }
                 }
             } else if prefPerformLA && laFailed {
-                retryAuthView()
-            }
-        }
-        .onChange(of: doAuthenticate) { doAuth in
-            if doAuth, authenticationExpired() {
-                runAuthenticate()
+                Button(action: runAuthenticate) { Label("Retry", systemImage: "arrow.counterclockwise") }
+            } else {
```

Comment:
Should `if prefPerformLA && laFailed` be split into separate ifs and its else branch only applied to `if laFailed`? Otherwise it's possible to offer authentication to a user who never set up the lock..